### PR TITLE
Update Installation-Windows.md correcting tensor flow version

### PR DESCRIPTION
**Pull request by sterlingcrispin**
edit for tensorflow 1.4 newer versions require cuda 9 which is unsupported by this repo
https://github.com/Unity-Technologies/ml-agents/issues/313

### CODE REVIEW

**Comment by vincentpierre** [comment]
Please make all PR to `develop` as mentioned in the [Contributing Guidelines](https://github.com/Unity-Technologies/ml-agents/blob/master/CONTRIBUTING.md)

**Comment by sterlingcrispin** [comment]
@vincentpierre changed

File: docs/Installation-Windows.md [hunk 1]
```diff
@@ -166,10 +166,10 @@ Make sure to replace the relevant directory location with the one you have insta
 </p>
 
 ### Install TensorFlow GPU
-Next, install `tensorflow-gpu` using `pip`. In an Anaconda Prompt with the Conda environment ml-agents activated, type in the following command _(make sure you are connected to the internet)_:
+Next, install `tensorflow-gpu` using `pip`. You'll need version 1.4.0 as newer version require CUDA 9 which is not yet supported. In an Anaconda Prompt with the Conda environment ml-agents activated, type in the following command _(make sure you are connected to the internet)_:
```
Comment (mmattar) [inline docs/Installation-Windows.md]: "as newer version" --> "as newer versions"

**Comment by mmattar** [comment]
@sterlingcrispin - we had one last review feedback above. We can merge as soon as it incorporated. Thanks!

**Comment by sterlingcrispin** [comment]
@mmattar hey got that updated

**Comment by mmattar** [comment]
Great - thanks @sterlingcrispin.

**Pull request merged**